packages/ui/src/stories/Markdown.stories.tsx

```
@@ -153,4 +165,27 @@
     },
 } as Meta<typeof Component>
 
+const MarkdownWithInteractiveComponents = ({ props }) => {
+
+    const markdownContent = props.children
+
+    const [markdownState, setMarkdownState] = useState(markdownContent)
+
+    return <>
+        <h1>Markdown:</h1>
+        <Component children={markdownState} onCheckboxChange={setMarkdownState}/>
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)